src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/CourtEvent.java

```diff
@@ -79,10 +79,10 @@
     @ManyToOne
     @NotFound(action = IGNORE)
     @JoinColumnsOrFormulas(value = {
-            @JoinColumnOrFormula(formula = @JoinFormula(value = "'" + EVENT_TYPE + "'", referencedColumnName = "domain")),
+            @JoinColumnOrFormula(formula = @JoinFormula(value = "'" + REASON + "'", referencedColumnName = "domain")),
```

Comment:
it turns out the codes for MOVE_RSN where pareet is CRN are a perfect match for values of EVENT_SUB_TYPE in T3